### ISSUE TITLE
ComposedPath Console Error in IE11 

### ISSUE THREAD

**Issue by pxwee5** [EducationLink/vue-tel-input#123]
![image](https://user-images.githubusercontent.com/14123790/62906053-352d2680-bdc1-11e9-9c23-d80117d93d25.png)

This is what I used in my project, no errors in IE11.
```
let elements = 
e.path ||
(e.composedPath ? e.composedPath() : false) ||
getParents(e.target);
```

**Comment by iamstevendao** [comment]
Hi @pxwee5, Thanks for your help! Would you be able to add a PR based on your experience?

**Comment by iamstevendao** [comment]
Fixed in [v3.1.4](https://github.com/EducationLink/vue-tel-input/releases/tag/v3.1.4)
